README.adoc

```diff
@@ -0,0 +1,75 @@
+[[minishift-addons]]
+= Minishift Add-ons
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
+
+toc::[]
+
+[[welcome-to-minishift]]
+== Welcome to the Minishift add-ons repository!
```

Comment:
Just Welcome provides no information to the user. the use of "add-ons" was preferred as the name of the repo is `minishift-addons`. It renders well for me...

![image](https://user-images.githubusercontent.com/1894/29066858-2574447c-7c63-11e7-9374-4167edf2df26.png)

But I would suggest to not use an exclamation mark if possible.